napalm/base/helpers.py

```diff
@@ -19,6 +19,7 @@
 import napalm.base.exceptions
 from napalm.base.utils.jinja_filters import CustomJinjaFilters
 from napalm.base.utils import py23_compat
+from napalm.base.interface_map import interface_map
```

Comment:
We shouldn't use the name `interface_map`  since there is already an existing `interface_map` in the ios.py code.